mineinabyss-components/src/main/kotlin/com/mineinabyss/components/armorstandlock/LockArmorStand.kt

```diff
@@ -0,0 +1,14 @@
+package com.mineinabyss.components.armorstandlock
+
+import com.mineinabyss.idofront.serialization.UUIDSerializer
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import java.util.*
+
+@Serializable
+@SerialName("mineinabyss:lockable")
+class LockArmorStand(
+    val owner: @Serializable(with = UUIDSerializer::class) UUID,
+    var lockState: Boolean = false,
+    var allowedAccess: List<@Serializable(with = UUIDSerializer::class) UUID?> = listOf(owner)
```

Comment:
Change this to a MutableSet, see info below on mutable, set is because order doesn't matter and they are faster